libbpf-tools/bindsnoop.h

```diff
@@ -12,7 +12,7 @@ struct bind_event {
 	int ret;
 	__u16 port;
 	__u8 opts;
-	__u8 proto;
+	__u16 proto;
```

Comment:
Please reorder this field to avoid padding.

Reply:
Normally it is done, but I got 64 bytes for this struct with and without reordering of this field.
I also removed the uneeded include of `maps.bpf.h`.